ui/app/pages/connected-sites/connected-sites.component.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from 'prop-types'
+import React, { Component } from 'react'
+import ConnectedSitesList from '../../components/app/connected-sites-list'
+import {
+  DEFAULT_ROUTE,
+} from '../../helpers/constants/routes'
+
+export default class ConnectSites extends Component {
+  static propTypes = {
+    history: PropTypes.object.isRequired,
```

Comment:
`withRouter` is not used with this component, so `history` won't be defined.

Reply:
weird, because this works...?

Reply:
Ah I see - the `ConnectedSites` container component is the top-level component for that route, so it's passed the route props from the `Route` component.